pkg/drivers/qemu/qemu.go

```diff
@@ -545,18 +550,21 @@ func (d *Driver) Start() error {
 }
 
 func hardwareAcceleration() string {
-	if detect.IsAmd64M1Emulation() {
-		return "tcg"
-	}
-	if runtime.GOOS == "darwin" {
-		// On macOS, enable the Hypervisor framework accelerator.
-		return "hvf"
-	}
 	if _, err := os.Stat("/dev/kvm"); err == nil && runtime.GOOS == "linux" {
 		// On Linux, enable the Kernel Virtual Machine accelerator.
 		return "kvm"
 	}
-	return ""
+	if runtime.GOOS != "darwin" {
+		return ""
+	}
+	if detect.IsAmd64M1Emulation() {
+		return "tcg"
+	}
+	if runtime.GOARCH == "arm64" && detect.GithubActionRunner() {
```

Comment:
Gemini tells me 
Flexibility: TCG allows QEMU to emulate a wide range of CPU architectures, even those not directly supported by the host system's hardware.   
 1.  Introduction — QEMU documentation 
[www.qemu.org](https://www.qemu.org/docs/master/system/introduction.html#:~:text=It%20supports%20a%20number%20of,capable%20of%20emulating%20many%20CPUs%2E)
Portability: Since TCG is software-based, QEMU can run on various platforms without requiring specialized hardware for each architecture.

so I am guessing this might mean we can use tcg on windows too

Reply:
Only for the slow path, which is **not** something that is recommended for running Kubernetes with

It should start with "whpx", and only use tcg if running on the wrong Architecture (like on Windows ARM)